cohttp-eio/src/cohttp_eio.mli

```diff
@@ -93,10 +93,20 @@ module Server : sig
 
   val run :
     ?socket_backlog:int -> ?domains:int -> port:int -> 'a env -> handler -> 'b
+  (** [run ~socket_backlog ~domains ~port env handler] runs a HTTP/1.1 server
+      executing [handler] and listening on [port]. [env] corresponds to
+      {!val:Eio.Stdenv.t}.
```

Comment:
Is this correct? `env` corresponds to the `env` above which only needs `domain_manager`, `net` and `clock` not all of `Eio.Stdenv.t`?

Reply:
Yes, not all of it. With that sentence, I was mostly trying to answer the question, "where do I get the instance of env?". Perhaps an example of `Server.run` will be more illustrative for that.